pset-iterators.js

```diff
@@ -170,9 +194,26 @@ called `shortTermPresidents`, declared with const
 console.log('Problem 6:')
 
 // Add your code below this line
-
+let presidentsOutOfOfficeArr = presidents.filter(function (element) {
```

Comment:
@kyleligman nice use of `.filter()` to get an array consisting only of former presidents